tests/unit/test_utils.py

```diff
@@ -99,3 +98,24 @@ def test_flatten_list(value: List, anticpated_result: List) -> None:
     """test for flatten list"""
     actual_result = utils.flatten_list(value)
     assert list(actual_result) == anticpated_result
+
+
+@pytest.mark.parametrize(
+    "value, anticpated_result",
+    [(1, "1s"), (60 + 1, "1m1s"), (3600 + 60 + 1, "1h1m1s"), (86400 + 3600 + 60 + 1, "1d1h1m1s")],
+    ids=["seconds", "minutes seconds", "hours minutes seconds", "days hours minutes seconds"],
+)
+def test_human_time(value: Union[int, float], anticpated_result: str) -> None:
+    """test for human time"""
+    # pass integer
+    int_result = utils.human_time(value)
+    assert int_result == anticpated_result
+    # pass negative integer
+    neg_int_result = utils.human_time(-value)
+    assert neg_int_result == f"-{anticpated_result}"
+    # pass float
+    float_result = utils.human_time(float(value))
+    assert float_result == anticpated_result
+    # pass negative float
+    neg_float_result = utils.human_time(-value)
+    assert neg_float_result == f"-{anticpated_result}"
```

Comment:
This looks like 4 distinct tests what would be better to either parameterize or separate.

Reply:
It was simple enough this should be fine.  Will split it out.